src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -128,9 +129,20 @@ class TezosNodeOperator(val node: TezosRPCInterface, batchConf: BatchFetchConfig
     * @param blockHash Hash of the block
     * @return          The `Future` list of operations
     */
-  def getAllOperationsForBlock(network: String, blockHash: BlockHash): Future[List[OperationGroup]] =
+  def getAllOperationsForBlock(network: String, blockHash: BlockHash): Future[List[TezosOperations.Group]] = {
+    import io.circe.parser.decode
+    import JsonDecoders.Circe.Operations._
```

Comment:
Is it possible to avoid wildcard imports? It makes the code harder to follow.

Reply:
This import is not used to enable visibility of names in the namespace, it's used to bring into scope all implicit decoders needed in scope.

To be honest I've never seen anyone explicitly importing the implicit names. This is how the pattern is meant to be used, to make implicits available